libs/components/behavioral/src/lib/landing-page-content/landing-page-content.component.scss

```diff
@@ -0,0 +1,29 @@
+:host {
+  display: block;
+  .metrics {
+    animation: fadeIn 2s;
+  }
+  .landing-page-content {
+    animation-duration: 2s;
+    animation-name: slidein;
+  }
+}
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+@keyframes slidein {
```

Comment:
During slide in you need to use relative (or maybe absolute?) positioning. Currently a big scroll bar appears during slide in

Reply:
I think that scroll bar appears because of the metrics component.